core/src/types.rs

```diff
@@ -72,6 +72,11 @@ impl<T> FromRc<T> for Arc<T> where T: Sized + Clone {
     }
 }
 
+//
+// Use Rc for values.
```

Comment:
This comment isn't great.  At first I thought it was just wrong, but now I see that this isn't an alias, it's picking one of `Rc<T>` and the implicitly not chosen `Arc<T>`.